Date_05/main.rb

```diff
@@ -0,0 +1,63 @@
+load 'customer.rb'
+require 'date'
+# require_relative 'connect_DB'
+
+class Main 
+  def insertCustomer
+    $cus_id = $client.query("SELECT MAX(cus_id) FROM customers").first["MAX(cus_id)"]   
+    @cus = Customer.new
+    @cus.input 
+    $client.query("insert into customers(name, gender, join_date) values ('#{@cus.username}', '#{@cus.gender}', '#{@cus.join_date}')")
+    print "Your ID is : #{$cus_id}\n" 
+  end 
+
+  def insertBill
+    @cus = Customer.new
+    @cus.buy($cus_id, $product_name, $quantity)
+    $client.query("insert into bills(cus_id, pro_name, quanlity) values (#{$cus_id.to_i}, '#{$product_name}', #{$quantity.to_i})")
+  end
+
+  def showProduct
+    print "Please enter your ID: "
+    $id_cus = gets.chomp.to_i
+    rs = $client.query("select pro_name, sum(quanlity) from bills  where cus_id = #{$cus_id} GROUP BY pro_name")
+    if (rs.count == 0) 
+      puts "You have not bought anything"
```

Comment:
Use single quote for normal string, double quote if we want insert variable to string.